<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecurityPasswordDialog</class>
 <widget class="QDialog" name="SecurityPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="CommonPushButton" name="pushButtonSecPwdReturn">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>479</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>210</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecPwdNum7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>220</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecPwdNum0">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>220</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecPwdNum4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>220</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecPwdShop">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>480</y>
     <width>220</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>#</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecPwdNum5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>220</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecPwdBack">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>27</y>
     <width>135</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退格</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecPwdNum2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>220</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>40</y>
     <width>411</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecPwdClear">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>369</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>210</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecPwdNum6">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>260</y>
     <width>220</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecPwdNum1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>220</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecPwdOK">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>259</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecPwdDot">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>220</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>22</y>
     <width>941</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
background-color: rgba(255, 255, 255, 128);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <zorder>pushButtonSecPwdBack</zorder>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecPwdNum8">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>370</y>
     <width>220</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecPwdNum3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>220</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecPwdNum9">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>370</y>
     <width>220</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPwdInfo">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>35</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>密码输入框</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>pushButtonSecPwdReturn</zorder>
  <zorder>pushButtonSecPwdNum7</zorder>
  <zorder>pushButtonSecPwdNum0</zorder>
  <zorder>pushButtonSecPwdNum4</zorder>
  <zorder>pushButtonSecPwdShop</zorder>
  <zorder>pushButtonSecPwdNum5</zorder>
  <zorder>pushButtonSecPwdBack</zorder>
  <zorder>pushButtonSecPwdNum2</zorder>
  <zorder>lineEditPassword</zorder>
  <zorder>pushButtonSecPwdClear</zorder>
  <zorder>pushButtonSecPwdNum6</zorder>
  <zorder>pushButtonSecPwdNum1</zorder>
  <zorder>pushButtonSecPwdOK</zorder>
  <zorder>pushButtonSecPwdDot</zorder>
  <zorder>pushButtonSecPwdNum8</zorder>
  <zorder>pushButtonSecPwdNum3</zorder>
  <zorder>pushButtonSecPwdNum9</zorder>
  <zorder>labelPwdInfo</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonSecPwdNum1</tabstop>
  <tabstop>pushButtonSecPwdNum2</tabstop>
  <tabstop>pushButtonSecPwdNum3</tabstop>
  <tabstop>pushButtonSecPwdNum4</tabstop>
  <tabstop>pushButtonSecPwdNum5</tabstop>
  <tabstop>pushButtonSecPwdNum6</tabstop>
  <tabstop>pushButtonSecPwdNum7</tabstop>
  <tabstop>pushButtonSecPwdNum8</tabstop>
  <tabstop>pushButtonSecPwdNum9</tabstop>
  <tabstop>pushButtonSecPwdDot</tabstop>
  <tabstop>pushButtonSecPwdNum0</tabstop>
  <tabstop>pushButtonSecPwdShop</tabstop>
  <tabstop>pushButtonSecPwdBack</tabstop>
  <tabstop>pushButtonSecPwdOK</tabstop>
  <tabstop>pushButtonSecPwdClear</tabstop>
  <tabstop>pushButtonSecPwdReturn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
